types/nodemailer/lib/mailer/index.d.ts

```diff
@@ -137,6 +137,7 @@ declare namespace Mail {
         dkim?: DKIM.Options;
         /** method to normalize header keys for custom caseing */
         normalizeHeaderKey?(key: string): string;
+        priority?: "high"|"normal"|"low";
```

Comment:
Seems like this should just be `"high"` or `"low"`, based off their source code

https://github.com/nodemailer/nodemailer/blob/master/lib/mailer/mail-message.js#L225...L238